FlashCard/FlashCard.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const FlashCard = (props) => {
+  const time = new Date()
```

Comment:
I would just remove this timestamp thing. Every time the card renders it shows a new `Date Created` which is not true.